mitosheet/src/mito/components/toolbar/FormulaTabContents.tsx

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Mito
+
+
+import React from 'react'
+
+import ToolbarButton from './ToolbarButton';
+import Dropdown from '../elements/Dropdown';
+import { ActionEnum, AnalysisData, EditorState, GridState, SheetData, UIState, UserProfile } from '../../types';
+import { MitoAPI } from '../../api/api';
+import { Actions } from '../../utils/actions';
+import DropdownItem from '../elements/DropdownItem';
+import { functionDocumentationObjects } from '../../data/function_documentation';
+import { getStartingFormula } from '../endo/celleditor/cellEditorUtils';
+
+export const FormulaTabContents = (
+    props: {
+        mitoAPI: MitoAPI
+        currStepIdx: number;
+        lastStepIndex: number;
+        highlightPivotTableButton: boolean;
+        highlightAddColButton: boolean;
+        actions: Actions;
+        gridState: GridState;
+        uiState: UIState;
+        setUIState: React.Dispatch<React.SetStateAction<UIState>>;
+        sheetData: SheetData;
+        userProfile: UserProfile;
+        editorState: EditorState | undefined;
+        setEditorState: React.Dispatch<React.SetStateAction<EditorState | undefined>>;
+        analysisData: AnalysisData,
+        sheetIndex: number,
+        closeOpenEditingPopups: () => void
+    }): JSX.Element => {
+
+    /**
+     * Maps the formula categories to the corresponding action enum
+     */
+    const formulaCategories: { [category: string]: ActionEnum } = {
```

Comment:
Can we make a `FuncitonCategory` type?

Reply:
- And then you could make `undefined` one valid type. 
- Then, you could add Formulas_Dropdown_More part of this map. 
- Then, you could avoid the special case for Formulas_Dropdown_More in render at the bottom.

Reply:
I don't think it's possible to make `undefined` a key in a dictionary but I added the type!